packages/react-ds/src/components/Button/index.spec.tsx

```diff
@@ -17,6 +17,11 @@ describe('<Button />', () => {
     expect(container.firstChild).toHaveClass('JSMButton--default')
   })
 
+  it('should have "primary" classname when "color prop" is "white"', () => {
```

Comment:
```suggestion
  it('should have "white" classname when "color prop" is "white"', () => {
```

Reply:
Thankss! Made in https://github.com/juntossomosmais/venice/pull/373/commits/15001534c672c479ad33d42d348492804b8930f3